.env.example

```diff
@@ -0,0 +1,4 @@
+# Do not report development environment errors to Sentry.io.
+SENTRY_DEVENV_NO_REPORT=1
```

Comment:
```suggestion
# SENTRY_DEVENV_NO_REPORT=1
```

Reply:
We want them to report by default.

Reply:
If @billyvg agrees I would port these here:
https://github.com/getsentry/bootstrap-sentry/blob/8c2cf34115b541034579e93b1104be02ed34a97d/bootstrap.sh#L423-L431
since `sentry` and `getsentry` read from the `.env` file.

Those variables are only being set for new engineers. Bringing them here would make it more likely that old guard would add it to their `.env` file.

I will leave it to Billy to say.

--------------------

@relaxolotl I think having the `.env` even with commented out variables is good since engineers *might* discover it and start using it.

Let's see what others think.

I personally have these:
```shell
❯ cat .env
SENTRY_INSTRUMENTATION=1
SENTRY_SILENCE_DIRENV_NOTICE=1
#SENTRY_DEVENV_NO_REPORT=1
SENTRY_SPA_DSN=https://863de587a34a48c4a4ef1a9238fdb0b1@o19635.ingest.sentry.io/5270453
# I do not want the post-merge step to auto-update dependencies
# SENTRY_POST_MERGE_AUTO_UPDATE=1
```